js/os.ts

```diff
@@ -23,6 +23,29 @@ interface CodeInfo {
   sourceMap: string | undefined;
 }
 
+interface IsTTY {
+  stdin: boolean;
+  stdout: boolean;
+  stderr: boolean;
+}
+
+/** Check if running in terminal.
+ *
+ * import { isTTY } from "deno";
+ * console.log(isTTY.stdout);
```

Comment:
Indent please - see examples in other exports of this file.

Reply:
also, isn't `console.log(isTTY().stdout);` ?